ddt.py

```diff
@@ -27,6 +27,8 @@
 DATA_ATTR = '%values'      # store the data the test must run with
 FILE_ATTR = '%file_path'   # store the path to JSON file
 UNPACK_ATTR = '%unpack'    # remember that we have to unpack values
+# Digits of the largest index for list/tuple, or default = 5 for generator
+I_LEN = len(str(len(DATA_ATTR))) if isinstance(DATA_ATTR, (list, tuple)) else 5
```

Comment:
I don't think this does what you think it does. This expression will always return a five, as it is computed statically upon module loading.

What you may want to do is compute this value dynamically, ideally based on the number of values that are provided via the data decorator.